src/app/contribute/guidelines/components/index.html

```diff
@@ -162,24 +162,34 @@
     During continuous integration builds, SKY UX runs visual regression tests through BrowserStack using <a href="http://www.protractortest.org/">Protractor</a> and <a href="https://github.com/koola/pix-diff">pix-diff</a>. These prerequisites are automatically installed when you run <stache-code>skyux e2e</stache-code> to create and compare screenshots in the <stache-code>skyux-spa-visual-tests/screenshots-baseline-local/</stache-code> folder.
   </p>
 
-  <p>
-    To create visual tests for a new component, create a folder for the tests in <stache-code>./src/app/visual</stache-code> and add four files for the visual regression test to the root <stache-code>e2e</stache-code> folder:
-  </p>
-
-  <ul>
-    <li>
-      <stache-code>{{ '{' }}componentName}-visual.component.html</stache-code>: The template for the component to render and take a screenshot of.
-    </li>
+  <ol>
     <li>
-      <stache-code>{{ '{' }}componentName}.e2e-spec.ts</stache-code>: The TypeScript code for the component to render and take a screenshot of.
+      To create visual tests for a new component, create a folder for the tests in <stache-code>./src/app/visual</stache-code> and add four files for the visual regression test to the root <stache-code>e2e</stache-code> folder:
```

Comment:
This should be
> ...create a folder for the tests in `./src/app/visual` and add **three** files to that new folder:

You do NOT need the bit about adding to the e2e folder.

Also, remove this line:

> {componentName}.e2e-spec.ts</stache-code>: The Protractor code to run the screenshot tests.

^ This particular file does not go in this folder, but instead it goes in the `./e2e` folder, which you already explain perfectly in the upcoming lines.